docs/pages/access-controls/guides/passwordless.mdx

```diff
@@ -164,7 +164,7 @@ cluster configuration:
 </Tabs>
 </TabItem>
 
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+<TabItem scope={["cloud"]} label="Teleport Enterprise">
```

Comment:
Same comment here as with the Locking guide: we should either remove these changes from the diff or show only the dynamic resources branch (and remove the tabs) to simplify the guide.